apps/web/src/AppRoutes.tsx

```diff
@@ -53,28 +54,30 @@ export const AppRoutes = () => {
 
   return (
     <Routes>
-      <Route path={ROUTES.AUTH_SIGNUP} element={<SignUpPage />} />
-      <Route path={ROUTES.AUTH_LOGIN} element={<LoginPage />} />
-      <Route path={ROUTES.AUTH_RESET_REQUEST} element={<PasswordResetPage />} />
-      <Route path={ROUTES.AUTH_RESET_TOKEN} element={<PasswordResetPage />} />
-      <Route path={ROUTES.AUTH_INVITATION_TOKEN} element={<InvitationPage />} />
-      <Route path={ROUTES.AUTH_APPLICATION} element={<QuestionnairePage />} />
-      <Route
-        path={ROUTES.PARTNER_INTEGRATIONS_VERCEL_LINK_PROJECTS}
-        element={
-          <EnsureOnboardingComplete>
-            <LinkVercelProjectPage type="create" />
-          </EnsureOnboardingComplete>
-        }
-      />
-      <Route
-        path={ROUTES.PARTNER_INTEGRATIONS_VERCEL_LINK_PROJECTS_EDIT}
-        element={
-          <EnsureOnboardingComplete>
-            <LinkVercelProjectPage type="edit" />
-          </EnsureOnboardingComplete>
-        }
-      />
+      <Route element={<AuthLayout />}>
```

Comment:
Wrap all of these paths with `AuthLayout` with an `<Outlet/>` added to `AuthLayout` to render children. We now remove `AuthLayout` being duplicated in each of these child routes.

Reply:
How about we call the `AuthLayout`, `PublicPageLayout` to make it more clear as Auth layout doesn't tell us if this is for authenticated or unauthenticated context.